hoomd_organics/base/system.py

```diff
@@ -492,6 +486,33 @@ def apply_forcefield(
             Neighborlist buffer for simulation cell.
 
         """
+        # make sure forcefield is xml based
+        self._validate_forcefield(force_field)
+
+        # check if molecules already had ff in them. If yes, use that  or
+        # return error (can't have two ff per molecule)
+
+        # Collecting all force-fields into a dict
+        if self._force_field:
```

Comment:
is `self._force_field` used anywhere outside of this function? Should we not have this as a class attribute anymore?